### DISCUSSION TITLE
Better Description Of Snippet And How To Customize It, My 2 Cents

### DISCUSSION THREAD

**Discussion by BeeGass** [allejo/jekyll-toc#57] [Ideas]
In the ReadMe there is a table that outlines the parameters needed which has been helpful for demystifying the customizing process but I have found that there is definitely area for improvement for documenting. 

1. I would like to recommend the use of more examples similar to the sister project [example here](https://github.com/allejo/jekyll-anchor-headings/wiki/Examples. )
2. I found the [blog post](https://allejo.io/blog/a-jekyll-toc-without-plugins-or-javascript/) by @allejo to be confusing. I think a more thorough guide could be beneficial for newbies in the front end world, like me, to better use this helpful snippet

**Comment by allejo** [comment]
I agree that having a Wiki on this repo would be a good idea just like the sister project has :+1:

Do you have specific topics in mind that you'd like to see covered in the Wiki?

**Reply by BeeGass** [reply]
I think for starters it would be helpful to define, with examples what each parameter does. A before and after if you will. That way people could have a solid understanding what they are getting into before they implement.
